coalib/output/printers/LOG_LEVEL.py

```diff
@@ -1,7 +1,12 @@
+import logging
 from coalib.misc.Enum import enum
+from colorama import Fore
 
-LOG_LEVEL = enum("DEBUG", "INFO", "WARNING", "ERROR")
-LOG_LEVEL_COLORS = {LOG_LEVEL.ERROR: "red",
-                    LOG_LEVEL.WARNING: "yellow",
-                    LOG_LEVEL.INFO: "blue",
-                    LOG_LEVEL.DEBUG: "green"}
+LOG_LEVEL = enum(DEBUG=logging.DEBUG,
+                 INFO=logging.INFO,
+                 WARNING=logging.WARNING,
+                 ERROR=logging.ERROR)
+LOG_LEVEL_COLORS = {LOG_LEVEL.ERROR: Fore.RED,
+                    LOG_LEVEL.WARNING: Fore.YELLOW,
+                    LOG_LEVEL.INFO: Fore.BLUE,
+                    LOG_LEVEL.DEBUG: Fore.GREEN}
```

Comment:
It may be not a good idea to separate this one in separate, cause we will have one commit after which there will be not a working version. Putting this change before or after the commit it is introduced in will break things